.github/workflows/develop.yml

```diff
@@ -279,6 +279,10 @@ jobs:
         with:
           # Shares cross multiple jobs
           shared-key: "coverage-test"
+      - name: Docker Cache
+        uses: ScribeMD/docker-cache@0.3.7
+        with:
+          key: docker-${{ runner.os }}-${{ hashFiles('tests-integration/fixtures/etcd/docker-compose-standalone.yml') }}
```

Comment:
since we have two compose files, we may need to find a better key strategy.

Reply:
@zyy17 @WenyXu any idea for this?

Reply:
can we use one key to store the etcd image cache?

Reply:
This cache plugin is designed to cache all images pulled/built after its setup. So it's not possible to cache etcd image only.

But after thinking about this, it's not very necessary to use file hash here for the key.